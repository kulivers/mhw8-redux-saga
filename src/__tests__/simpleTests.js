function sum(x, y) {
  return x + y;
}
function mockFn(fn, x, y) {
  return fn(x, y);
}

describe('default tests', () => {
  // beforeAll(() => {
  //   console.log('before all');
  // });
  it('toEqual', () => {
    expect(sum(2, 3)).toEqual(5.0);
  });
  it('toBe', () => {
    expect(sum(2, 3)).toBe(5);
  });
  it('toMatchSnapshot', () => {
    expect(sum(2, 3)).toMatchSnapshot();
  });
  it('toBeGreaterThan', () => {
    expect(sum(2, 3)).toBeGreaterThan(1);
  });
  it('Mocking function', () => {
    const jestFn = jest.fn((x, y) => x + y);
    expect(mockFn(jestFn, 1, 1)).toBe(2);
    expect(jestFn).toBeCalledWith(1, 1);
  });
  it('not', () => {
    expect(1).not.toBe(2);
  });
});
