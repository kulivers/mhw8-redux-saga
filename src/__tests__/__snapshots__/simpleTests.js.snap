// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default tests toMatchSnapshot 1`] = `5`;
